rtlib/hash_stats.hh

```diff
@@ -1,5 +1,5 @@
-#ifndef HASH_STATS
-#define HASH_STATS
+#ifndef RTLIB_HASH_STATS_HH_
```

Comment:
Why is it considered a better style to add underscores at the end?

Reply:
honestly: I don't know and I cannot find anything about it in Google's styleguide: https://google.github.io/styleguide/cppguide.html#Header_Files
But hey, as long as it doesn't break anything we can comply to this rule